### PR TITLE
Debug/hs

### PR DESCRIPTION
**Pull request by anthonycorletti**
> **Please do not create a pull request without creating an issue first.**
>
> Changes need to be discussed before proceeding, pull requests submitted without linked issues may be rejected.
>
> Please provide enough information so that others can review your pull request. You can skip this if you're fixing a typo – it happens.

* [ ] I have added tests to `tests` to cover my changes.
* [ ] I have updated `docs/`, if necessary.
* [ ] I have updated the `README.md`, if necessary.

***What existing issue does this pull request close?***

Put `closes #issue-number` in this pull request's description to auto-close the issue that this fixes.

***How are these changes tested?***

This pull request includes automated tests for the code it touches and those tests are described below. If no tests are included, reasons why must be provided below.

These changes are tested with [...]

***Demonstration***

Demonstrate your contribution.

For example, what are the exact commands you ran and their output, related screenshots, screen-recordings, test runs, anything that can showcase.

***Provide additional context.***

Provide as much relevant context as you like.

### CODE REVIEW

File: dataquality/utils/dq_logger.py [hunk 1]
```diff
@@ -99,3 +101,28 @@ def get_dq_log_file(
...
+    Attempts to remove a directory and all its contents.
+
+    Args:
+        dir_path (str): the path to the directory to remove
+    """
+    import shutil
+    import time
+
+    retry = 0
+    while retry < 3:
```
Comment (Ben-Epstein) [inline dataquality/utils/dq_logger.py:118]: What happens when retry hits 3? Do we skip deleting?

**Review by Ben-Epstein — COMMENTED** [review]
Q inline

File: dataquality/utils/dq_logger.py (hunk 1 above)
Comment (anthonycorletti) [reply dataquality/utils/dq_logger.py:118]: We've already attempted deletion. One thing we could do is warn the user if any content still exists  after num_retries.

Comment (anthonycorletti) [reply dataquality/utils/dq_logger.py:118]: Wdyt?

Comment (Ben-Epstein) [reply dataquality/utils/dq_logger.py:118]: Yea fair enough. We do full deletion at the end, and it's not a persistent environment anyway

**Review by Ben-Epstein — APPROVED** [review]

**Pull request merged**